### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by jbampton**
### Description

This PR fixes some spelling

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

### CODE REVIEW

File: plugins/user-authenticators/ldap/src/main/java/org/apache/cloudstack/ldap/LdapConfiguration.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public class LdapConfiguration implements Configurable{
             String.class,
             "ldap.user.memberof.attribute",
             "memberof",
-            "the reverse membership attibute for group members",
+            "the reverse membership attribute for group members",
```
Comment (DaanHoogland) [inline plugins/user-authenticators/ldap/src/main/java/org/apache/cloudstack/ldap/LdapConfiguration.java:68]: this one will not update on existing installations and should (very striuctly speaking) have an update statement in the schema upgrade file.

File: server/src/main/java/com/cloud/configuration/Config.java [hunk 2]
```diff
@@ -228,7 +228,7 @@ public enum Config {
             String.class,
             "network.loadbalancer.haproxy.stats.visibility",
             "global",
-            "Load Balancer(haproxy) stats visibilty, the value can be one of the following six parameters : global,guest-network,link-local,disabled,all,default",
+            "Load Balancer(haproxy) stats visibility, the value can be one of the following six parameters : global,guest-network,link-local,disabled,all,default",
```
Comment (DaanHoogland) [inline server/src/main/java/com/cloud/configuration/Config.java:231]: these as well won´t be updated in the db on existing installations

File: server/src/main/java/com/cloud/configuration/Config.java [hunk 3]
```diff
@@ -1696,7 +1696,7 @@ public enum Config {
             String.class,
             "baremetal.ipmi.lan.interface",
             "default",
-            "option specified in -I option of impitool. candidates are: open/bmc/lipmi/lan/lanplus/free/imb, see ipmitool man page for details. default valule 'default' means using default option of ipmitool",
+            "option specified in -I option of impitool. candidates are: open/bmc/lipmi/lan/lanplus/free/imb, see ipmitool man page for details. default value 'default' means using default option of ipmitool",
```
Comment (DaanHoogland) [inline server/src/main/java/com/cloud/configuration/Config.java:1699]: and this (db will remain the same)

**Review by DaanHoogland — COMMENTED** [review]
@jbampton you must know the source by head now ;)
this change looks generally good to me. The only comment is about the descriptions of global settings, these should probably be changed in the DB as well, though this is not a definite -1 to the change

File: test/integration/component/maint/test_redundant_router.py [hunk 4]
```diff
@@ -1641,7 +1641,7 @@ def test_06_updateVRs_in_sequence(self):
             if retry == 0:
                 break
             time.sleep(self.testdata["sleep"])
-            self.info("wating for backup router to become primary router name:"+new_backup_router.name)
+            self.info("waiting for backup router to become primary router name:"+new_backup_router.name)
```
Comment (stephankruggg) [inline test/integration/component/maint/test_redundant_router.py:1644]:
```suggestion
            self.info("waiting for backup router to become primary router name: "+new_backup_router.name)
```

File: test/integration/component/test_haproxy.py [hunk 5]
```diff
@@ -850,7 +850,7 @@ def test_10_create_policy_enable_disable_vpn(self):
 
     @attr(tags=["advanced", "advancedns"], required_hardware="false")
     def test_11_invalid_params(self):
-        """Test verfify functionality syncronous and asyncronous validations"""
+        """Test verify functionality syncronous and asyncronous validations"""
```
Comment (stephankruggg) [inline test/integration/component/test_haproxy.py:853]:
```suggestion
        """Test verify functionality synchronous and asynchronous validations"""
```

File: test/integration/component/test_haproxy.py [hunk 6]
```diff
@@ -815,7 +815,7 @@ def test_10_create_policy_enable_disable_vpn(self):
         # 1. create an account
         # 2. using that account,create an instances
```
Comment (stephankruggg) [inline test/integration/component/test_haproxy.py:816]:
```suggestion
        # 2. using that account, create an instance
```

File: test/integration/component/test_haproxy.py [hunk 7]
```diff
@@ -815,7 +815,7 @@ def test_10_create_policy_enable_disable_vpn(self):
...
         # 3. select the Source NAT IP  and configure the stikiness policy
```
Comment (stephankruggg) [inline test/integration/component/test_haproxy.py:817]:
```suggestion
        # 3. select the Source NAT IP and configure the stickiness policy
```

File: test/integration/component/test_haproxy.py [hunk 8]
```diff
@@ -815,7 +815,7 @@ def test_10_create_policy_enable_disable_vpn(self):
...
-        # 4. enable /disable the VPN. It should not impact the ceated rules
+        # 4. enable /disable the VPN. It should not impact the created rules
```
Comment (stephankruggg) [inline test/integration/component/test_haproxy.py:818]:
```suggestion
        # 4. enable/disable the VPN. It should not impact the created rules
```

File: test/integration/component/test_haproxy.py [hunk 9]
```diff
@@ -815,7 +815,7 @@ def test_10_create_policy_enable_disable_vpn(self):
...
         #    listLBStickinessPolicies Api should show created stikiness policy
```
Comment (stephankruggg) [inline test/integration/component/test_haproxy.py:819]:
```suggestion
        #    listLBStickinessPolicies Api should show created stickiness policy
```

File: test/integration/component/test_haproxy.py [hunk 10]
```diff
@@ -850,7 +850,7 @@ def test_10_create_policy_enable_disable_vpn(self):
 
     @attr(tags=["advanced", "advancedns"], required_hardware="false")
     def test_11_invalid_params(self):
-        """Test verfify functionality syncronous and asyncronous validations"""
+        """Test verify functionality syncronous and asyncronous validations"""
 
         # Validate the following
         #  verify the validation  while creating or attaching stikiness policy
```
Comment (stephankruggg) [inline test/integration/component/test_haproxy.py:856]:
```suggestion
        # verify the validation while creating or attaching stickiness policy
```

File: CHANGES.md [hunk 11]
```diff
@@ -682,7 +682,7 @@ Bug Fixes:
 
 * CLOUDSTACK-1600 Typo in dpkg-buildpackage command
 * CLOUDSTACK-1574 updateResourceCount API is failed saying to specify valida resource type even after parsing the valid resource type
-* CLOUDSTACK-1562 Replace the short-cut solution of supportting @DB with the formal one
+* CLOUDSTACK-1562 Replace the short-cut solution of supporting @DB with the formal one
```
[the remark this answers is not in the document]
Comment (DaanHoogland) [reply CHANGES.md:685]: wouldn´t "short cut" be more 'English'? i.e. no dash and no concatination

**Comment by DaanHoogland** [comment]
@stephankruggg , though I see the point of most of your additions, shouldn´t these be a separate PR? cc @jbampton .
I don´t mind, but if we try to make the world perfect in one PR we will fail.

File: CHANGES.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (stephankruggg) [reply CHANGES.md:685]: I'm not really sure, both alternatives are widely used. However, "shortcut" seems to be the one that's found in most dictionaries, so I think it's the best option here.

File: ui/public/locales/zh_CN.json [hunk 12]
```diff
@@ -3331,7 +3331,7 @@
   "message.enabling.zone": "\u6B63\u5728\u542F\u7528\u8D44\u6E90\u57DF",
   "message.enabling.zone.dots": "\u6B63\u5728\u542F\u7528\u8D44\u6E90\u57DF...",
 
-  "message.enter.seperated.list.multiple.cidrs": "\u5982\u679C\u5B58\u5728\u591A\u4E2A CIDR\uFF0C\u8BF7\u8F93\u5165\u7528\u9017\u53F7\u5206\u9694\u7684 CIDR \u5217\u8868",
+  "message.enter.separated.list.multiple.cidrs": "\u5982\u679C\u5B58\u5728\u591A\u4E2A CIDR\uFF0C\u8BF7\u8F93\u5165\u7528\u9017\u53F7\u5206\u9694\u7684 CIDR \u5217\u8868",
```
Comment (BryanMLima) [inline ui/public/locales/zh_CN.json:3334]: I do not think this is used anywhere, there is no entry for this in the `en.json` file as well. You could just remove it.

File: CHANGES.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (BryanMLima) [reply CHANGES.md:685]: @stephankruggg I agree, `shortcut` looks like the right one, I always use [Merriam-Webster](https://www.merriam-webster.com/dictionary/shortcut) as source and there is no entry for `short-cut`.

**Comment by DaanHoogland** [comment]
@jbampton you have a conflict. Can you have a look? Can you also comment on whether you want to cooperate on this PR or rather see smaller PRs be merged, faster.

**Comment by jbampton** [comment]
@DaanHoogland pull requests take a long time to be merged.

I have 4 that are not merged

**Comment by DaanHoogland** [comment]
> @DaanHoogland pull requests take a long time to be merged.
> 
> I have 4 that are not merged

yes, I know. We have a requirement that we have 2 LTGM and a succesfull (regression) test. That is why I didn´t want to much extra changes on this PR.
I'll run the tests but I still have that remark about upgrading.
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
no consistent errors related to this PR for the test_vpc_vpn errors see #6512

**Pull request merged**